Projects/Team3/Smart_House/Smart_House/AirConditioner.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Smart_House
+{
+    class AirConditioner : Device
+    {
+
+        private int _delta;
+        public AirConditioner(int delta = 1)
+        {
+            this._delta = delta;
+        }
+
+        public void Handler(object o, EventArgs e)
```

Comment:
it could be great to have unit tests for methods

Reply:
We are  going to try it)